project/config/settings/base.py

```diff
@@ -257,7 +257,10 @@
             "level": "DEBUG",
             "class": "logging.StreamHandler",
             "formatter": "verbose",
-        }
+        },
+        "null": {
```

Comment:
Can you explain what this does? (Sorry if it is obvious and I missed it)

Reply:
In general, f6eddc6 is declares a logger that throws away log messages and then says "use this logger for everything under test"

I originally put that there when I was running tests with the django runner to capture log messages under test because they were all getting printed to stdout, but didn't clock that once I switched them over to running with pytest, pytest does this for us in a more configurable way. I've backed this change out in cb766dd. We should rely on pytest's capturing: https://docs.pytest.org/en/latest/capture.html
`pytest` will capture the logs by default and `pytest --capture=no` (or `pytest -s`) won't, which is more useful anyway.

I've done cb766dd as a rever commit to make review easier, but I can tidy up history before merge